target/linux/qualcommax/image/Makefile

```diff
@@ -25,16 +25,15 @@ define Device/FitImageLzma
 endef
 
 define Device/EmmcImage
-	IMAGES += factory.bin sysupgrade.bin
+	IMAGES := factory.bin sysupgrade.bin
 	IMAGE/factory.bin := append-rootfs | pad-rootfs | pad-to 64k
 	IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-to 64k | sysupgrade-tar rootfs=$$$$@ | append-metadata
 endef
 
 define Device/UbiFit
 	KERNEL_IN_UBI := 1
-	IMAGES := factory.ubi sysupgrade.bin
```

Comment:
Sorry but no, why are you modifying the default recipe?

Reply:
It's already defined in `Device/Default`: https://github.com/openwrt/openwrt/blob/main/target/linux/qualcommax/image/Makefile#L10-L11

Reply:
Ok, I get the idea now.

But can you please split the IMAGES changes and DEVICE_PACKAGES into 2 commits and please extend the commit descriptions as it is not really clear to me what this commit is trying to do now

Reply:
Updated.
I'm not sure if we really need to use ubi image for devices where we use other factory image:

- cmcc_rm2-6
- netgear_rax120v2
- netgear_wax630
- yuncore_ax880